web/assets/music.js

```diff
@@ -0,0 +1,31 @@
+import { MUSIC_ENABLED } from "../config.js";
+
+let playing = false
+
+export const startMusic = () => {
+  const music = document.getElementById("loadscreen-music");
+  if ( MUSIC_ENABLED ) {
+    playing = true
+    music.volume = 0.1;
+    document.addEventListener('keypress', (event) => {
+      if(event.key == "p") {
+        if(music.paused) {
+          music.play();
+        }else{
+          music.pause();
+        }
+      }else if(event.key == "s") {
```

Comment:
Change these coercive equalities (==) to the strict type equality check, por favor (===)

Reply:
Ok